autovalet_gazebo/launch/aruco_sim.launch

```diff
@@ -0,0 +1,32 @@
+<!-- To run simulation correctly along with this launch file:
+Download the package aruco_gazebo for the markers from here: https://github.com/joselusl/aruco_gazebo.git
+We import aruco_visual_marker_7 model in our world
+Detection done using aruco_ros package from pal-robotics: https://github.com/pal-robotics/aruco_ros.git 
+Put the packages in ~catkin_ws/src/-->
```

Comment:
are those info still legit for us? if we're not using the `aruco_visual_marker_7` model, we should either remove those descriptions or revise it accordingly to avoid future confusion.

Reply:
They are up-to-date. But this stuff is already in the readme, so I can remove it from here.